common/src/types/proof_bundles.rs

```diff
@@ -107,11 +105,12 @@ pub struct GenericValidCommitmentsBundle<
 }
 
 /// A type alias that specifies the default generics for `GenericValidCommitmentsBundle`
-pub type ValidCommitmentsBundle = GenericValidCommitmentsBundle<MAX_BALANCES, MAX_ORDERS, MAX_FEES>;
+pub type ValidCommitmentsBundle =
+    Box<GenericValidCommitmentsBundle<MAX_BALANCES, MAX_ORDERS, MAX_FEES>>;
 
 /// A bundle of the statement, witness commitment, and proof of `VALID MATCH MPC`
 #[derive(Clone, Debug, Serialize, Deserialize)]
-pub struct ValidMatchMpcBundle {
+pub struct GenericValidMatchMpcBundle {
```

Comment:
nit, but should this be prefixed `Generic` when it doesn't take any generic parameters?

Reply:
what name would read better? I went with `Generic` to maintain consistency in the relationship between this and analogous types